docs/benchmark.md

```diff
@@ -36,25 +36,30 @@ Custom workloads can also be benchmarked with SOCI.
 
 In order to run the benchmarks on custom workloads the custom container image needs to have its soci indices generated and pushed to a contianer registry as described in the [Getting started docs](/docs/getting-started.md)
 
-Generate benchmark binaries: 
+Generate benchmark binaries:
 ``` make build-benchmarks``` will generate benchmark binaries for performance testing and comparison testing against overlayFS. The binaries will be available in the ```/benchmark/bin``` folder.
- 
-### Benchmark binaries have different cli flags: 
+
+### Benchmark binaries have different cli flags:
 
 | Flag | Description | Required / Optional |
 |----------|----------|----------|
-| ```-f``` | File path to a csv file containing details of multiple images to be tested | Optional
+| ```-f``` | File path to a json file containing details of multiple images to be tested | Optional
 | ```-count``` | Specify number of times the benchmarker needs to run  | Optional
 | ```-show-commit``` | Tag the latest commit hash to the benchmark results | Optional
 
-We can now run benchmarks on custom workloads using the ```-f``` flag to specify the file path to a csv file containing details of the workloads.
+We can now run benchmarks on custom workloads using the ```-f``` flag to specify the file path to a json file containing details of the workloads.
 
-### Csv file format for custom workloads
+### JSON file format for custom workloads
 
 Ensure that the file being used with the ```-f``` flag follows the following format
 
 ```shell
-<Test_name>, <Container_Image_ref>, <Ready_line>, <Soci_Index_Manifest_Digest>
+{
+    "short_name": "<Test_name>"
+    "image_ref": "<Container_Image_ref>"
+    "ready_line": "<Ready_line>"
+    "soci_index_digest": "<Soci_Index_Manifest_Digest>"
+}
 ```
 
 Example :
```

Comment:
Lets update the example for the JSON format.

Reply:
👍